app/screens/TalkDetailsScreen/AssistantsList.tsx

```diff
@@ -34,7 +34,7 @@ export function AssistantsList(props: AssistantsListProp) {
               <View style={$assistantLinks}>
                 <IconButton
                   icon={assistant.twitter ? "twitter" : "link"}
-                  onPress={() => openLinkInBrowser(assistant.twitter || assistant.externalURL)}
```

Comment:
This property doesn't exist anymore, at least from the response as of writing this. However, some speakers do have a website